runtime/doc/options.txt

```diff
@@ -3903,7 +3782,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	When on, the keyboard is mapped for the Hebrew character set.
 	Normally you would set 'allowrevins' and use CTRL-_ in insert mode to
 	toggle this option.  See |rileft.txt|.
-	NOTE: This option is reset when 'compatible' is set.
```

Comment:
:+1: These "when compatible set" lines obviously should be removed, because 'compatible' cannot be set anymore.